images/build-e2e/lib/darwin/filter.xsl

```diff
@@ -0,0 +1,19 @@
+<xsl:stylesheet version="1.0" 
```

Comment:
Are those xsls files exactly the same? If that is the case we need to have it only once and add it from the Containerfile

Reply:
Ok, I was rounding the idea. Will rework the PR